agent/util-scripts/pbench-tool-meister-start

```diff
@@ -1,33 +1,52 @@
 #!/usr/bin/env python3
 # -*- mode: python -*-
 
-"""pbench-tool-meister-start
+"""pbench-tool-meister-start - start the execution of the Tool Meister
+sub-system.
 
-Responsible for orchestrating the Redis Server, Tool Data Sink (TDS), and Tool
-Meisters (TMs):
+There are two roles tool-meister-start plays:
+
+  1. (optionally) orchestrate the creation of the instances of the Redis
+     server, Tool Data Sink, and Tool Meisters
+
+  2. execute the start sequence for the Tool Meister sub-system
+
+The `--orchestrate=(create|existing)` command line parameter is used to control
+the orchestration.  The default is to "create" the instances of the Redis
+server, Tool Data Sink, and Tool Meisters.  If the user specifies "existing",
+then the command assumes the `--redis-server` and `--tool-data-sink` parameters
+will be provided to direct the command to the location of those instances.
+
+The sequence of steps to execute the above behaviors is as follows:
 
    1. Loading tool group data for the requested tool group
-   2. Starting a Redis server and creating the Redis channel for the TDS to
-      talk to the client
+      - This is the first step regardless so that the tool information can be
+        validated, the number of TMs and their marching orders can be
+        enumerated, and when orchestrating the TMs, the list of where those
+        TMs are requested to run can be determined
+   2. [optional] Starting a Redis server and creating the Redis channel for
+      the TDS to talk to the client
```

Comment:
Only the "starting" is optional, right?  Creating the channel is done regardless of who started the server, right?  (Why does this make me nervous....)